src/test/java/org/isf/permissions/TestGroupPermissionManager.java

```diff
@@ -53,4 +61,20 @@ void testFindByIdIn() throws Exception {
 		assertThat(groupPermissionList).isNotEmpty();
 		assertThat(groupPermissionList).hasSize(2);
 	}
+
+	@Test
+	@DisplayName("Update group permissions")
+	void updateGroupPermissions() throws Exception {
+		UserGroup userGroup = userGroupManager.findByIdIn(List.of("admin")).stream().findFirst().orElseGet(() -> null);
+		List<Integer> permissionsIds = List.of(10, 11, 12);
+		assertThat(userGroup).isNotNull();
+		List<GroupPermission> groupPermissions = groupPermissionManager.findUserGroupPermissions(userGroup.getCode());
+		List<Permission> permissions = groupPermissionManager.update(userGroup, permissionsIds, false);
+		assertThat(permissions).isNotEmpty();
+		assertThat(permissions).hasSize(groupPermissions.size());
```

Comment:
These assertions don't really illustrate the group permission replacement. It would be more precise if before updating (not replacing) the group permissions, the target group already has some assigned permissions. And then after updating , you have to check that: 
- new permissions didn't replace the old ones
- already assigned permissions are not re-assigned (duplicates)
- non existing permissions are skipped
- the number of the group's permissions in the number of old assigned permissions and new assigned ones

Reply:
+1

Reply:
Done from line 76